tfx/dependencies.py

```diff
@@ -61,6 +61,7 @@ def make_pipeline_sdk_required_install_packages():
       'packaging>=20,<21',
       'portpicker>=1.3.1,<2',
       'protobuf>=3.13,<4',
+      'strip-hints>=0.1.0<1',
```

Comment:
I don't think we need this anymore after Python 2 support was dropped?

Reply:
The reason to keep it is that hints may require an arbitrary artifact class (ex: OutputArtifact[MyDataset]). 

Which means we would need to programatically detect those and insert import statements for each one of them.

Given that typehints are not useful for execution is better to just strip them away here for simplicity and also for speed of execution (we dont need to incur the cost of importing non execution classes)